File: 029-Encode the Number/EncodetheNumber.py
def encode_number(N):
    encoded = ''
    str_N = str(N)
    for i in range(len(str_N)):
        digit = int(str_N[i])
        square = digit * digit
        encoded += str(square)
    return int(encoded)

# Example usage:
input1 = 34
print(encode_number(input1))  # Output: 916

File: 029-Encode the Number/EncodetheNumber.cpp
#include<iostream>
#include<string>
using namespace std;

int encodeNumber(int N){
    string strN = to_string(N);
    string result = " ";

    for(int i = 0; i < strN.length(); i++){
        int digit = strN[i] - '0';
        int square = digit * digit;
        result += to_string(square);
    }

    return stoi(result);
}
int main(){
    int N = 34;
    cout<<encodeNumber(N) << endl; // Output: 916
    return 0;
}